src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -279,4 +280,10 @@ export class WorkbasketFiltersComponent implements OnInit {
     this.selected.caseType = undefined; // option should be blank rather than "Select a value" in case of reset.
     this.selectedJurisdictionCaseTypes = null;
   }
+
+  private getCaseFields() {
+    if (this.workbasketInputs) {
+      this.caseFields = this.workbasketInputs.map(item => FieldsUtils.convertToCaseField(item.field));
+    }
+  }
```

Comment:
looks like there is a duplicated method in src/shared/components/search-filters/search-filters.component.ts, can this extract to a common method in common ts?

Reply:
It's not a duplicated method. Search filters is referencing searchInputs. Both components are extremely similar and there was talk about combining the two because they are so similar (copy, paste and a tweak by the look of it) but it was decided to keep them separate.

Reply:
But you could still extract it(the referencing) as a param(input) in a common method, then you don't need this new private method but can call/assign the result directly within the method in ngOnInit().

Reply:
I don't think this is a big deal really. They're _similar_ but all the methods are actually doing is a bit of null checking and a quick filter. When I first looked, I assumed they were identical but they're not and I don't really think it's worth the effort of refactoring them into a util method after all. If there were more instances, I might change my mind... but it's only two. 🤷‍♂️